task/04-arrays-tasks.js

```diff
@@ -536,7 +670,11 @@ function selectMany(arr, childrenSelector) {
  *   [[[ 1, 2, 3]]], [ 0, 0, 1 ]      => 2        (arr[0][0][1])
  */
 function getElementByIndexes(arr, indexes) {
-    throw new Error('Not implemented');
+   
+   return eval('arr[' + indexes.toString().replace(/\s/g, "").replace(/,/g, "][") + ']');
```

Comment:
To solve this task, it is more correct to process the array and not convert array to string for parsing